qCC/ccColorFromScalarDlg.cpp

```diff
@@ -459,18 +452,20 @@ void ccColorFromScalarDlg::onApply()
 					col[i] = 1.0f - col[i];
 				}
 			}
-
+				
 			//calculate and set colour
 			ccColor::Rgb rgb = ccColor::Convert::hsv2rgb(col[0] * 360.0f, col[1], col[2]);
-			m_cloud->setPointColor(p, ccColor::FromQColor(QColor(int(rgb.r), int(rgb.g), int(rgb.b), int(col[3] * 255))));
+			m_cloud->setPointColor(p, ccColor::FromQColor(QColor(
```

Comment:
You can directly use ccColor::Rgba(rgb, static_cast<ColorCompType>(col[3] * ccColor::MAX))